src/collector/WebBackEndCollector.spec.js

```diff
@@ -0,0 +1,117 @@
+import WebBackEndCollector from './WebBackEndCollector.js';
+
+describe('WebBackEndCollector', () => {
+  const originalEvent = {
+    id: '11bf5b37-e0b8-42e0-8dcf-dc8c4aefc000',
+    clientId: '11bf5b37-e0b8-42e0-8dcf-dc8c4aefc001',
+    eventType: 'state',
+    eventAction: 'load',
+    eventSource: 'products/shoes/1',
+    eventCategory: 'products/shoes',
+    deviceType: 'mobile'
+  };
+
+  const newPerfAPIResponse = [
+    {
+      responseStart: 555
```

Comment:
@robertdumitrescu COVERAGE - This is omitting the fact that the values are rounded.

Reply:
Good point.

Reply:
DONE